src/components/ComposerAttachments.vue

```diff
@@ -422,7 +432,19 @@ export default {
 <style scoped lang="scss">
 
 .new-message-attachments {
-	ul.new-message-attachments--list {
+	&--counter {
+		color: var(--color-text-maxcontrast);
+		padding: 10px 20px;
+		cursor:pointer;
+		display:flex;
+		align-items: center;
+
+		&--with-errors {
+			color:red;
+		}
+	}
+
```

Comment:
please place ```cursor:pointer;``` to all child elements
`	&--counter * {
		cursor:pointer;
	}
`

Reply:
Done in https://github.com/nextcloud/mail/pull/7066/files#diff-13360562f049327b0c3be2784d54548428fbe349d2a0a43a04a5278104998722R247